Movecraft/src/main/java/net/countercraft/movecraft/commands/CraftInfoCommand.java

```diff
@@ -48,51 +48,34 @@ public static void registerProvider(@NotNull Function<Craft, String> provider){
         providers.add(provider.andThen(List::of));
     }
 
-
-    @Override
-    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
-        if(args.length == 0){
-            if (!(sender instanceof Player)){
-                sender.sendMessage("Supply a parameter.");
-                return true;
-            }
-            Craft craft = CraftManager.getInstance().getCraftByPlayer(((Player) sender));
-            if(craft == null){
-                sender.sendMessage("You must be piloting a craft.");
-                return true;
-            }
-            craftInfo(sender, craft, 1);
-            return true;
-        }
-        OptionalInt pageQuery;
-        if (sender instanceof  Player && (pageQuery = MathUtils.parseInt(args[0])).isPresent()){
-            Craft craft = CraftManager.getInstance().getCraftByPlayer(((Player) sender));
-            if(craft == null){
-                sender.sendMessage("You must be piloting a craft.");
-                return true;
-            }
-            craftInfo(sender, craft, pageQuery.getAsInt());
-            return true;
-        }
-        var craft = CraftManager.getInstance().getCraftByPlayerName(args[0]);
+    @Default
+    @Syntax("[player|self] <page>")
```

Comment:
Can we remove the "self" argument from commands?  I'd like it to default to yourself like previous.

Reply:
aight